docs/best-practices/documentation.md

```diff
@@ -50,16 +61,33 @@ For projects hosted primarily in BitBucket, any of these templates may be copied
 
 ### Additional documentation
 
-Additional documentation should be located in `./docs`.
+Additional documentation should be located in `./docs/`.
 
-[Docsify](https://docsify.js.org/#/) is a nice static site generator for documentation.
+### Features
 
-See this repository's `docs/index.html` for a usage example.
+We're starting to use Gherkin for describing user-facing functionality, and have learning about best practices from a couple of resources:
```

Comment:
"...describing user-facing functionality, and have learning about"
probably should have this be:
"...describing user-facing functionality, and we're learning..."